Update outdated information (#192)
infinum/wordpress-handbook: Coding standards/CSS coding standards.md

We use the same rules for writing CSS as defined in the [frontend handbook](https://infinum.com/handbook/frontend/SASS%20Styleguide/File%20organization), just like with JavaScript coding standards.

> The new WordPress projects use [Tailwind CSS](https://tailwindcss.com/docs). Some older projects use [Sass](http://sass-lang.com/) and [BEM](http://getbem.com/) methodology to write HTML.

Reviewer: ```suggestion
The new WordPress projects use [Tailwind CSS](https://tailwindcss.com/docs). Some older projects use [Sass](http://sass-lang.com/) for styling, with [BEM](http://getbem.com/) naming.
```
